pkg/kubelet/eviction/eviction_manager_test.go

```diff
@@ -347,6 +382,109 @@ func TestMemoryPressure_VerifyPodStatus(t *testing.T) {
 	}
 }
 
+func TestPIDPressure_VerifyPodStatus(t *testing.T) {
+	testCases := map[string]struct {
+		wantPodStatus v1.PodStatus
+	}{
+		"eviction due to pid pressure": {
+			wantPodStatus: v1.PodStatus{
+				Phase:   v1.PodFailed,
+				Reason:  "Evicted",
+				Message: "The node was low on resource: pids. Threshold quantity: 1200, available: 500. ",
+			},
+		},
+	}
+	for name, tc := range testCases {
+		for _, enablePodDisruptionConditions := range []bool{true, false} {
+			t.Run(fmt.Sprintf("%s;PodDisruptionConditions=%v", name, enablePodDisruptionConditions), func(t *testing.T) {
+				defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.PodDisruptionConditions, enablePodDisruptionConditions)()
+
+				podMaker := makePodWithPIDStats
+				summaryStatsMaker := makePIDStats
+				podsToMake := []podToMake{
```

Comment:
These names don't make much sense for PIDs. We don't count pids as a resource so there is no "below" or "above" request here. Please remove.